.github/workflows/build.yml

```diff
@@ -48,9 +48,15 @@ jobs:
           - "clang"
         os:
           - "macos-10.15"
+          - "macOS-11"
```

Comment:
small nitpick since it trigged me. would be nice to have macOS for both in the same case, if it isn't case sensitive.

Reply:
Right, didn't even notice I wrote it like that last night :D .

Fixed.